mods/weather/init.lua

```diff
@@ -56,16 +56,17 @@ minetest.register_globalstep(
 	    local weathertype = weather_pr:next(0, 19)
 
 	    -- on avg., every 1800 frames the weather.weather will change to one of:
-	    -- 13/20 chance of clear weather
-	    -- 6/20 chance or stormy weather
+	    -- 15/20 chance of clear weather
+	    -- 3/20 chance or stormy weather
 	    -- 1/20 chance of snowstorm weather
 
-	    if weathertype < 13 then
+	    if weathertype < 15 then
 	       weather.weather = "clear"
 	    elseif weathertype < 19 then
 	       weather.weather = "storm"
 	    elseif weathertype < 20 and snow_enable then
 	       weather.weather = "snowstorm"
+	    else weather.weather = "clear"
```

Comment:
Ok thanks rubenwardy, I'll change this line in my pull request.
And I'll try to make a single pull request for every commit that I've done too, so it will be easier to merge.